protocol/testing/mainnet/mainnet.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -eo pipefail
+
+# This file initializes a non-validating full node for mainnet
+
+source "./vars.sh"
+
+CHAIN_ID="dydx-mainnet-1"
+
+# Define the mapping from version to URL
+declare -A version_to_url
+# version_to_url["v5.1.0"]="https://github.com/dydxprotocol/v4-chain/releases/download/protocol%2Fv5.1.0-dev4/dydxprotocold-v5.1.0-dev4-linux-amd64.tar.gz"
```

Comment:
Doesn't this still need to be defined?

Reply:
No, since mainnet is on v5.1.x we are using the current commit for v5.1.0 in this PR